scripts/gulpfiles/build_tasks.js

```diff
@@ -733,7 +733,8 @@ function cleanBuildDir() {
 function format() {
   return gulp.src([
     'core/**/*.js', 'core/**/*.ts',
-    'blocks/**/*.js', 'blocks/**/*.ts'
+    'blocks/**/*.js', 'blocks/**/*.ts',
+    '.eslintrc.js'
```

Comment:
If you can't get clang-format to behave itself feel free to remove this and just fix the formatting manually.

Reply:
Got it to work via moving the JSON logic to a `const` away from `module.exports`